common/elasticsearch/elasticsearch.go

```diff
@@ -27,13 +27,15 @@ import (
 )
 
 const (
-	ESIndex = "heapster"
+	ESIndex   = "heapster"
+	ESCluster = "default"
 )
 
 type ElasticSearchService struct {
 	EsClient      *elastic.Client
 	bulkProcessor *elastic.BulkProcessor
 	baseIndex     string
+	Cluster       string
```

Comment:
I think we should call it `ClusterName` and not just cluster.. it'll be easier to understand imo